Exercise_A/assets/css/style.css

```diff
@@ -1,3 +1,158 @@
 /* 
 Add in your custom style such that you can develop the wonderful website within the exercise. 
 */
+
+html,body {
+    width: 100%;
+    scroll-behavior: smooth;
+    text-align: center;  
+    position: relative;
+}
+
+h1,h2,h3,h4,h5,h6 {
+    color: #fff;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+    :root {
+      scroll-behavior: smooth;
+    }
+  }
+
+body {
+    font-family: 'Helvetica Neue',Arial,sans-serif;
+    font-size: 14px;
+    line-height: 1.43;
+}
+
+/* --- navbar ---  */
+
+.navbar {
+    background-color: #fff;
+}
+
+
+.linebreaker {
+        height: 1rem;
+        max-width: 3.25rem;
+        margin: 1rem auto;
+        border: 3px solid #f05f40;
+        background-color: #f05f40;
+        opacity: 1;
+      }
+
+/*  --- main header --- */
+
+header {
+    padding-top: 20% 20%;
+    position: relative;
+    width: 100%;
+    height: 100%;
+    min-height: auto;
+    text-align: center;
+    color: #fff;
+    background-image: url(/FTWD_Web_Day_04/Exercise_A/assets/images/header.jpg);

```

Comment:
style.css is located inside css folder so you need to go back one directory and head into the image folder
background-image: url("../images/header.jpg");